nebula/ui/components/MZSettingsToggle.qml

```diff
@@ -58,12 +56,6 @@ CheckBox {
         }
     ]
 
-    MZToolTip {
```

Comment:
Just curious, why remove the tooltip?

Reply:
All of our toggles have labels next to them, so the tooltips were unnecessary. similarly none of our checkboxes had tooltips